Attempt to "fix" cursor wobbling by increasing hysteresis detection timeout.

See https://bugs.freedesktop.org/show_bug.cgi?id=104828 for an explanation.

--- libinput-1.10.0/src/evdev-mt-touchpad.c
+++ libinput-1.10.0/src/evdev-mt-touchpad.c
@@ -134,10 +134,10 @@
 static inline void
 tp_maybe_disable_hysteresis(struct tp_dispatch *tp, uint64_t time)
 {
-	/* If the finger is down for 80ms without seeing motion events,
+	/* If the finger is down for 120ms without seeing motion events,
 	   the firmware filters and we don't need a software hysteresis */
 	if (tp->nfingers_down >= 1 &&
-	    time - tp->hysteresis.last_motion_time > ms2us(80)) {
+	    time - tp->hysteresis.last_motion_time > ms2us(120)) {
 		tp->hysteresis.enabled = false;
 		evdev_log_debug(tp->device, "hysteresis disabled\n");
 		return;
